templates/definition/charger/homematic.yaml

```
@@ -0,0 +1,65 @@
+template: homematic
+products:
+  - brand: Homematic eQ-3 AG (ELV)
```

Auf deren Webseite verwenden sie diese Markenbezeichnung.
 
```suggestion
  - brand: Homematic IP
```